src/scss/components/_tooltips.scss

```diff
@@ -0,0 +1,131 @@
+.hint {
+  cursor: help;
+  display: inline-block;
+  font-style: normal;
+  font-weight: normal;
+  letter-spacing: normal;
+  position: relative;
+  text-transform: none;
```

Comment:
Are all of these styles required for this to function? It seems like we should just need `cursor` and `position` — I just want to make sure this class is useful (and doesn’t override styles that aren’t related to tooltips).